utils/vlog/tests/dsp_combinational/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_file_target(FILE dsp_combinational.sim.v SCANNER_TYPE verilog)
 v2x_test_both(NAME dsp_combinational TOP_MODULE dsp_combinational)
+vpr_test_pbtype(NAME dsp_combinational)
```

Comment:
I'm okay with vpr_test_pbtype being part of the v2x_test_both function

Reply:
Done.